integration-test/src/main/java/com/sequenceiq/it/cloudbreak/testcase/e2e/distrox/DistroXStopStartScaleTest.java

```diff
@@ -56,6 +56,13 @@ public void testStopStartScaleDistroX(TestContext testContext, ITestContext iTes
         AtomicReference<List<String>> instancesToStop = new AtomicReference<>();
         DistroXScaleTestParameters params = new DistroXScaleTestParameters(iTestContext.getCurrentXmlTest().getAllParameters());
 
+        long expectedNumber;
+        if (testContext.commonCloudProperties().getCloudProvider().equalsIgnoreCase("AZURE")) {
```

Comment:
can we move this to the application.yml ? In that case we can easily find the property in the future

Reply:
long expectedNumber this field in application.yml right?